src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <article class="calendar">
```

Comment:
Please add this modifiers for calendar instead of 'calendar day'
![image](https://github.com/mate-academy/layout_calendar/assets/54936044/0dee0f12-7d25-4368-ba37-59331c7ee739)